File: src/leetcode/python/SingleNumber_136.py
class SingleNumber_136(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int

        n ^ n = 0
        the last value will be the only one
        """
        return 2 * sum(set(nums)) - sum(nums)

File: src/leetcode/java/SingleNumber_136.java
package leetcode.java;

public class SingleNumber_136 {
  // n ^ n = 0
  // the last value will be the only one
  public int singleNumber(int[] nums) {
    int ans = 0;
    for (int i = 0; i < nums.length; i++) {
      ans ^= nums[i];
    }
    return ans;
  }
}
